x-pack/plugins/maps/public/connected_components/toolbar_overlay/set_view_control/set_view_form.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Component } from 'react';
+import { EuiButtonEmpty, EuiPopover, EuiRadioGroup } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { MapCenter, MapSettings } from '../../../../common/descriptor_types';
+import { DecimalDegreesForm } from './decimal_degrees_form';
+import { MgrsForm } from './mgrs_form';
+import { UtmForm } from './utm_form';
+
+const DEGREES_DECIMAL = 'dd';
+const MGRS = 'mgrs';
+const UTM = 'utm';
+
+const COORDINATE_SYSTEM_OPTIONS = [
+  {
+    id: DEGREES_DECIMAL,
+    label: i18n.translate('xpack.maps.setViewControl.decimalDegreesLabel', {
+      defaultMessage: 'Degrees Decimal',
```

Comment:
Is this correct? I've only ever used "[Decimal degrees](https://en.wikipedia.org/wiki/Decimal_degrees)". Maybe both are valid. 🤷

Reply:
It should be "decimal degrees" thanks